src/util/webauthn.js

```diff
@@ -68,25 +68,29 @@ function isWebAuthnSupported() {
 
 
 // eslint-disable-next-line no-unused-vars
-function isPlatformAuthenticatorSupported() {
+async function isPlatformAuthenticatorSupported() {
   if (window.PublicKeyCredential === undefined ||
     typeof window.PublicKeyCredential !== 'function' ||
     typeof window.PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable !== 'function') {
     return true;
   }
-
-  if (window.PublicKeyCredential) {
-    // eslint-disable-next-line no-undef
-    PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable()
-      .then(function(available){
-        return available;
-      }).catch(function(err){
-      // Something went wrong
-        console.error(err);
-      });
+  if( !await isWebAuthnSupported()) {
```

Comment:
`isWebAuthnSupported` is not async.
I think you can replace the line 77 to 93 with the following:
   if (isWebAuthnSupported()) {
      return await PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable();
    }
    return false;